### PR TITLE
IGAPP-850: iOS notification permission handling

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

Basically just gets rid of throwing an exception and uses return value instead to reset the shown settings. Sadly it is not possible to open apps notification settings directly. Maybe we could show some hint though before opening settings which should be discussed/designed here: https://issues.tuerantuer.org/browse/IGAPP-854

### CODE REVIEW

**Review by Taggotty — APPROVED** [review]
Uhhm. Tested on Android emulator ^^

File: translations/translations.json [hunk 1]
```diff
@@ -2401,20 +2401,8 @@
...
       "offersDescription": "Jobsuche, Praktikasuche und mehr in der Umgebung",
-      "inquiryTitle": "Einstellungen",
-      "inquiryIntro": "Durch Klicken auf \"{{accept}}\" erlaube ich der {{appName}}-App, ...",
-      "sentryTitle": "App-Stabilität",
-      "sentryCondition": "... mit dem automatischen Senden von Absturzberichten {{appName}} zu verbessern",
-      "pushNewsTitle": "Push-Benachrichtigungen",
-      "pushNewsCondition": "... mir Push-Benachrichtigungen zu den neuesten lokalen Nachrichten zu senden",
       "skip": "Überspringen",
-      "next": "Weiter",
-      "cancel": "Abbrechen",
```
Comment (SteffiStuffel) [inline translations/translations.json:2412]: Do we really want to delete these translations, its really generic stuff we may need sometime in the future?

**Review by SteffiStuffel — APPROVED** [review]
Not tested as I cannot test ios-stuff right now.

File: translations/translations.json (hunk 1 above)
Comment (klinzo) [reply translations/translations.json:2412]: If we need them we can either translate them again or we look in the history and revert the commit. No need to keep them.

**Pull request merged**